apiary.apib

```diff
@@ -1214,7 +1214,7 @@ Only project members with Admin or Share roles, members of the maintainer, membe
 
 
 ### List project Metadata [GET]
-Only project members with Admin or Share roles, members of the maintainer, members of organization and super admins can list metadata.
+Only project members, members of the maintainer, members of organization and super admins can list metadata.
```

Comment:
pokud to dobře dole čtu, tak musí mít uživatel MFA aby to mohl číst (bez ohledu na roli). To mi tady chybí

Reply:
To sa imho vztahuje jak keby k inej funkcnosti (to ze je tam nejake obmedzenie na MFA), v https://keboolamanagementapi.docs.apiary.io/#reference/projects/project-metadata nemame nic take spomenute a tiez tam funguju podobne obmedzenia.

Reply:
to ze se to resi i jinde nebo ze to jinde taky nemame ale neni duvod proc to tady uzivateli nedat. Jasne, pri prvnim pokusu to zjisti v Exception, ale proc ho tomu vystavovat. Dal bych za to (pokud to je pravda) "All users listing these metadata have to have the MFA enabled"

Reply:
Ono ide o to, ze je to ina funkcnost/validacia/cast dokumentacie ktora toto riesi `All users listing these metadata have to have the MFA enabled` toto nieje pravda pretoze, nemusis mat zapnute MFA ak to nevyzaduje org. az ked to vyzaduje org. ze musis mat zapnute MFA tak ti to nepojde, ako na ostatnych miestach v KBC